### ISSUE TITLE
Maven dependency is wrong

### ISSUE THREAD

**Issue by adamtwiss** [maxmind/GeoIP2-java#43]
In the Readme
```
   <dependency>
        <groupId>com.maxmind.geoip2</groupId>
        <artifactId>geoip2</artifactId>
        <version>v2.3.0</version>
    </dependency>
```
The version bit should be just '2.3.0' and not 'v.2.3.0'

**Comment by 2shortplanks** [comment]
Thanks for pointing this out Adam, you're quite right.  We'll plan a point
release after the holiday weekend (i.e. on Monday) to address this.

**Comment by 2shortplanks** [comment]
Fixed in 2.3.1
